src/components/CommentForm.jsx

```diff
@@ -0,0 +1,8 @@
+
+const CommentForm = () => {
+  return (
+    <div>CommentForm</div>
+  )
+}
+
+export default CommentForm
```

Comment:
I'm slightly confused, it doesn't seem like you have a form to add a new comment